apps/client/resources/public/stylesheets/main.css

```diff
@@ -467,6 +469,20 @@ ul#sites-available > li > a {
   line-break: anywhere;
 }
 
+div.missing-keys {
+  background-color: black;
```

Comment:
We should consider adding white and black as vars(e.g. `--black: #000` and then called with `var(--black)`).  It is a little overkill, though it allows us to adjust our white/black values according to the palette of the rest of the site, and not have to adjust all instances throughout the css.

In general, it is a bad code smell for me to have explicit colours in css rules, and prefer them set to vars.

Reply:
Global values lead to consistency. I've updated this is 9ff053f